solutions/projects-stack-da/README.md

```diff
@@ -0,0 +1 @@
+# Sample IBM Projects stack
```

Comment:
```suggestion
# Sample deployable architecture stack 
```

Or if we need to get at this is being created in projects, "Sample deployable architecture stack from an IBM Cloud Project"